src/types/query.rs

```diff
@@ -0,0 +1,64 @@
+use edn_rs::Serialize;
+
+#[derive(Clone)]
+pub struct Query {
+    find: Vec<String>,
+    whr: Option<Vec<String>>,
```

Comment:
Why simplified `where`? 🤔

Reply:
reserved keyword

Reply:
Ohhh I forgot about that 🤦‍♂️ 
Hmm okay then, I could only think about `_where` or `where_` instead of `whr`, but I don't think is a big change/advantage.

Reply:
the field is not readable, so where_ seems ok.

Reply:
b9d493b